src/Function_2.py

```
@@ -0,0 +1,107 @@
+CAMERA_INDEX = 0
+OUTPUT_FOLDER = "auto_captured_images"
+
+# Đường dẫn đến file XML của Haar Cascade
+HAAR_CASCADE_PATH = "haarcascade_frontalface_default.xml"
+
+
+# Khoảng thời gian nghỉ (cooldown) giữa các lần chụp (tính bằng giây)
+# Hệ thống sẽ chỉ chụp 1 ảnh mỗi 1 giây nếu vẫn thấy khuôn mặt
+CAPTURE_COOLDOWN = 1  # 1 giây
+
+def auto_capture_on_detect():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)